packages/alfa-style/src/property/will-change.ts

```diff
@@ -0,0 +1,48 @@
+import { Parser } from "@siteimprove/alfa-parser";
+import {
+  CustomIdent,
+  Keyword,
+  List,
+  type Parser as CSSParser,
+} from "@siteimprove/alfa-css";
+
+import { Longhand } from "../longhand.js";
+
+const { either } = Parser;
+
+type Specified =
+  | Keyword<"auto">
+  | List<Keyword<"scroll-position"> | Keyword<"contents"> | CustomIdent>;
+type Computed = Specified;
+
+const illegalCustomIdents = [
+  "will-change",
+  "auto",
+  "scroll-position",
+  "contents",
+];
+
+const animatableFeature: CSSParser<
+  Keyword<"scroll-position"> | Keyword<"contents"> | CustomIdent
+> = either(
+  Keyword.parse("scroll-position", "contents"),
+  CustomIdent.parse(
+    (customIdent) => !illegalCustomIdents.includes(customIdent.value),
+  ),
```

Comment:
```suggestion
  CustomIdent.parse(not(illegalCustomIdents.includes)),
```
should work with the change in `CustomIdent`.
(it doesn't read super well, OTOH, so maybe keeping it as `customIdent => !illegalCustomIdents.includes(customIdent)` is still better… Or we add a `String.isInArray` predicate, since we do this quite a lot 🤔 Maybe for later…

Reply:
It does look nicely succinct, but I think we would also need to instantiate and store the illegal custom idents and not as strings, i.e. `const illegal = [ CustomIdent.of("will-change"), ...]`. I'll leave this one for later :)

Reply:
No, if `CustomIdent.parse` is updated to take a `Predicate<string>` as per https://github.com/Siteimprove/alfa/pull/1707#discussion_r1839804289, that should work 🤔

Reply:
Oh, sorry, I missed that comment 😖 Then it makes sense 👍🏼